mne/connectivity/envelope.py

```diff
@@ -107,22 +126,28 @@ def envelope_correlation(data, combine='mean', orthogonalize="pairwise",
         corr = np.empty((n_nodes, n_nodes))
         for li, label_data in enumerate(epoch_data):
             if orthogonalize is False:  # the new code
-                label_data_orth = data_mag
-                label_data_orth_std = data_mag_std
+                label_data_orth = data_mag[li]
+                label_data_orth_std = data_mag_std[li]
             else:
                 label_data_orth = (label_data * data_conj_scaled).imag
+                np.abs(label_data_orth, out=label_data_orth)
```

Comment:
First change: when orthogonalizing, use `abs` of the orthogonalized data rather than just `(data * data_conj_scaled).imag)`. Not 100% sure this is right but conceptually the thing we're correlating with is `abs(data)` so it seems to make sense to take the `abs` here, too...

I think the original operated on `np.log10(label_data_orth ** 2)` so an `abs` was implicitly taken by the squaring operation.

Reply:
I am not sure about the log10, but they do square and take the log-transform:
Citation from their supplementary information:
_"The  signal  Y(t,f)  is  orthogonalized  in  the  complex  plane  with respect to X(t,f). This results in a positive number |Y_orth_X(t,f)| which is then squared and log-transformed, and correlated with [|X(t,f)|."_ [Source](https://static-content.springer.com/esm/art%3A10.1038%2Fnn.3101/MediaObjects/41593_2012_BFnn3101_MOESM9_ESM.pdf)

Reply:
log10 is the same as log up to a scale factor, and that scale factor goes away in the correlation so it doesn't really matter. But `np.log` is faster than `np.log10` so why not!

Reply:
... but that text does make it clear that it should result in a positive number even before the square and log transform, so I think this is correct now (and will continue to be on the next push where I change `np.log10` to `np.log`)

Reply:
I think abs was missing in the original code, because of the squaring in the next step.